modules/magento/repositories/useUrlResolverRepository.ts

```diff
@@ -7,15 +7,17 @@ export default () => {
   return async (
     url: string,
   ): Promise<{
-    id: string
+    entityUid: string
+    id: number
```

Comment:
Id should stay as string (if Magento returns it as number, convert it to string). Numbers are used for values that will be used in computations (addition, subtraction, ...). If you have data that you do not expect to perform any mathematical computation on it (e.g. IDs), treating it as string, even though its actually a number is generally a better idea.

Reply:
We should also mark the id as deprecated (Magento marked it as deprecated already)

Reply:
fixed